src/api/api.js

```diff
@@ -4,11 +4,20 @@ const newsApi = axios.create({
   baseURL: "https://dh-ncnews.cyclic.app/api",
 });
 
-export const getArticles = async () => {
-  const res = await newsApi.get("/articles");
+export const getArticles = async (topic_slug) => {
+  let path = "/articles";
+  if (topic_slug) {
```

Comment:
I'd look at maybe using the request config for axios here. A quick Google should take you to the docs 👍